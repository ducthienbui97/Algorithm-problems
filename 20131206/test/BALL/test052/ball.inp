7 746089 541506 135159 314037
103146 597111 764794 406742 414220 246851 664767 958149 748477 808168 67161 239321 660249 507419 777441 689673 274493 101463 666868 155106 146626 99790 741211 751573 453560 965841 601224 191181 103920 333143 664358 597368 374552 226269 933461 586316 509480 578719 497546 307440 283621 143819 888836 22630 274092 127157 958456 485863 12472 
716971 413691 101735 244706 95220 5876 969598 430052 653074 69965 82806 170489 975499 706930 870976 377466 997137 424893 292840 298403 947163 627059 439320 622793 561005 986509 458602 748568 414451 847856 955811 603232 948506 649617 335190 400640 901254 332437 567299 853299 691002 403254 176804 409732 77986 597832 533866 725315 814146 
190079 475708 216161 854960 292131 466915 874671 368331 756250 11929 948105 185497 998980 908719 932405 331165 312501 998263 991007 829633 345453 313577 757108 10858 705406 667662 518101 28958 378577 745263 820535 486662 923541 746283 934608 761764 450510 726212 106376 220039 963322 598659 958929 902012 281051 285796 155937 985837 907795 
65202 447299 839296 228357 796871 854735 72536 916555 904186 570402 310462 729400 217144 898640 617802 727798 140259 115203 205168 90342 585466 33517 992310 423289 311009 666046 879394 213245 752288 424689 551729 969045 327766 996881 234213 140013 665775 299574 499113 649527 213349 953296 629338 920427 443268 365769 315435 217599 592729 744865 716568 116542 141751 947419 786818 681662 342272 381795 270035 457275 538334 249901 749244 346995 116415 730801 523216 261366 596687 119231 258587 963784 466072 735892 262415 66931 644900 201790 405102 203043 801737 946087 61665 779240 656751 375147 755667 437644 389161 843683 332773 392803 497946 140081 121723 54348 966763 230084 20714 36777 781209 29109 878508 742297 373233 179794 938939 706452 575884 529885 212805 623105 11043 352935 220159 439734 839634 631010 321625 94826 623533 335049 757339 662465 282131 363910 266396 723679 764401 941999 436132 726943 374842 240866 97700 950027 164785 598952 772611 740610 496785 465204 615081 198342 704124 341427 458382 542976 536020 640032 536466 197219 174110 391486 689341 775256 350210 594273 164138 296520 976111 275885 874063 555982 790693 518411 262364 583850 901037 370153 246376 593891 227813 699895 148014 102419 823033 54679 861998 780449 138462 704568 805059 846017 456795 393131 636003 593504 435802 572852 383661 43155 64691 849083 202630 532385 313110 35904 432342 574158 281272 78538 8644 510780 571455 173765 877052 61260 720504 34041 947806 511138 803709 621483 220327 429236 839988 724592 227346 857174 880260 584835 304560 94729 770918 472798 481392 502280 176439 450520 244087 655430 765176 526601 168762 651364 540410 103502 904667 417506 605950 872946 411128 544221 85542 320317 222979 339397 175578 352801 412725 410018 724866 236224 571193 170806 743075 939688 120421 235457 921092 399313 177723 449169 261696 794451 93341 817159 966918 750139 6275 451073 856179 847871 457313 386717 353750 852377 811640 612458 3816 998124 92378 977432 846640 745532 607607 419983 956705 717030 839057 853163 396756 697188 625455 920478 270515 994547 553556 920428 903166 497185 947932 325331 205391 790518 859274 405238 415738 45109 181754 460681 92233 89089 463541 908627 953058 452318 969572 632104 943633 807135 74720 987271 509174 773456 835882 856030 397858 744146 386258 250361 15759 522090 766357 983666 667093 770518 283738 19144 382997 837314 321903 