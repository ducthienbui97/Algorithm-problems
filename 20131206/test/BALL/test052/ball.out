9740802895903
